codecs/opus/src/main/jni/OpusJni.vcxproj

```diff
@@ -0,0 +1,143 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this file be part of .gitignore?

Reply:
It is needed for Windows native library build. It can be removed later if Windows is not going to be supported.